src/integ-test/bdd-test/features/primitives/paGetPaymentRes_syntax_ko.feature

```diff
@@ -0,0 +1,288 @@
+       <soapenv:Header />
+       <soapenv:Body>
+          <paf:paGetPaymentRes>
+             <outcome>OK</outcome>
+             <data>
+                <creditorReferenceId>$iuv</creditorReferenceId>
+                <paymentAmount>10.00</paymentAmount>
+                <dueDate>2021-12-31</dueDate>
+                <!--Optional:-->
+                <retentionDate>2021-12-31T12:12:12</retentionDate>
+                <!--Optional:-->
+                <lastPayment>1</lastPayment>
+                <description>description</description>
+                <!--Optional:-->
+                <companyName>company</companyName>
+                <!--Optional:-->
+                <officeName>office</officeName>
+                <debtor>
+                   <uniqueIdentifier>
+                      <entityUniqueIdentifierType>G</entityUniqueIdentifierType>
+                      <entityUniqueIdentifierValue>77777777777</entityUniqueIdentifierValue>
+                   </uniqueIdentifier>
+                   <fullName>paGetPaymentName</fullName>
+                   <!--Optional:-->
+                   <streetName>paGetPaymentStreet</streetName>
+                   <!--Optional:-->
+                   <civicNumber>paGetPayment99</civicNumber>
+                   <!--Optional:-->
+                   <postalCode>20155</postalCode>
+                   <!--Optional:-->
+                   <city>paGetPaymentCity</city>
+                   <!--Optional:-->
+                   <stateProvinceRegion>paGetPaymentState</stateProvinceRegion>
+                   <!--Optional:-->
+                   <country>IT</country>
+                   <!--Optional:-->
+                   <e-mail>paGetPayment@test.it</e-mail>
+                </debtor>
+                <!--Optional:-->
+                <transferList>
+                   <!--1 to 5 repetitions:-->
+                   <transfer>
+                      <idTransfer>1</idTransfer>
+                      <transferAmount>10.00</transferAmount>
+                      <fiscalCodePA>77777777777</fiscalCodePA>
+                      <IBAN>IT45R0760103200000000001016</IBAN>
+                      <remittanceInformation>testPaGetPayment</remittanceInformation>
+                      <transferCategory>paGetPaymentTest</transferCategory>
+                   </transfer>
+                </transferList>
+                <!--Optional:-->
+                <metadata>
+                   <!--1 to 10 repetitions:-->
+                   <mapEntry>
+                      <key>1</key>
+                      <value>22</value>
+                   </mapEntry>
+                </metadata>
+             </data>
+          </paf:paGetPaymentRes>
+       </soapenv:Body>
+    </soapenv:Envelope>
+    """
+    And <elem> with <tagvalue> in paGetPayment
+    And if outcome is KO set fault to None in paGetPayment
```

Comment:
@davideromagnolieveris is this step correct?
`fault` tag is not present in the the paGetPayment XML

Reply:
Wrong XML paGetPayment; insert the following:
`"""
    <soapenv:Envelope xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/" xmlns:paf="http://pagopa-api.pagopa.gov.it/pa/paForNode.xsd">
       <soapenv:Header/>
       <soapenv:Body>
          <paf:paGetPaymentRes>
             <outcome>#outcome#</outcome>
             <fault>
                <faultCode>#faultCode#</faultCode>
                <faultString>#faultString#</faultString>
                <id>#id#</id>
                <description>#description#</description>
             </fault>
          </paf:paGetPaymentRes>
       </soapenv:Body>
    </soapenv:Envelope>
    """`